libc/src/math/daddf128.h

```diff
@@ -0,0 +1,22 @@
+//===-- Implementation header for daddf128 --------------------------*- C++
+//-*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_SRC_MATH_DADDF128_H
+#define LLVM_LIBC_SRC_MATH_DADDF128_H
+
+#include "include/llvm-libc-types/float128.h"
```

Comment:
We used to include libc/src/__support/macros/properties/types.h instead of including float128.h directly.